### PR TITLE
SVG support

### PR DESCRIPTION
**Pull request by pravdomil**
Patch to support SVG using https://wordpress.org/plugins/svg-support/

### CODE REVIEW

File: js/src/views/edit-attribute-field-attachment.js [hunk 1]
```diff
@@ -103,12 +103,13 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
...
-				attachment.sizes.thumbnail :
-				_.first( _.sortBy( attachment.sizes, 'width' ) );
+			attachmentThumb = (
+				typeof attachment.sizes !== 'undefined' &&
+				typeof attachment.sizes.thumbnail !== 'undefined'
+			) ? attachment.sizes.thumbnail : _.first( _.sortBy( attachment.sizes, 'width' ) ) || {};
 
 			jQuery( '<img/>', {
-				src:    attachmentThumb.url,
+				src:    attachmentThumb.url || attachment.url,
```
Comment (goldenapples) [inline js/src/views/edit-attribute-field-attachment.js:112]: It looks like this is still going to be throwing a lot of warnings about trying to get non-existant properties ('url', 'width', 'height') of `attachmentThumb` when it's an empty object.

Comment (pravdomil) [inline js/src/views/edit-attribute-field-attachment.js:112]: please post me the errors, I got nothing wrong

Comment (goldenapples) [reply js/src/views/edit-attribute-field-attachment.js:112]: I was mistaken, there aren't any errors here. :+1:

**Review by goldenapples — APPROVED** [review]
Looks good, though I'd like to see a comment explaining why we're doing this.

**Comment by pravdomil** [comment]
This is basic support for https://wordpress.org/plugins/svg-support/ plugin

**Comment by mattheu** [comment]
I think a more generic comment would be good. `Support for images with alternate image 'sizes'.`

**Comment by goldenapples** [comment]
Closing this PR, this is fixed in #654.

**Pull request closed without merging**